### PR TITLE
Ishu Suggestions to Instructions

### PR DESCRIPTION
**Pull request by ishuverma**
Still working on some of the images but wanted to do the first PR so you can start looking at what I've so far

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+This demo showcases Intelligent IoT Gateway on the Red Hat OpenShift Container Platform. The Intelligent IoT Gateway enables real time decision-making at the edge. OpenShift provides secure enterprise-grade container platform to accelerate IoT application delivery. Through the use of containers and microservices, this demo enables security, scalability and ease of deployment.
+
+image::images/IoT-OSE-Containers.png[width="462", height="614", align=center]
+
+*Please note: This repository is currently under development and the feature set/components are subject to change without notice*
+
+== Components
+
+* https://access.redhat.com/documentation/en/red-hat-xpaas/version-0/red-hat-xpaas-fuse-integration-services-image/[Fuse Integration Services]*
```
Comment (sabre1041) [inline README.adoc]: Replace \* with - in bullet point

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+Red Hat JBoss Fuse Integration Services provides a set of tools and containerized xPaaS images that enable development, deployment, and management of integration microservices within OpenShift.
+* https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-a-mq-image/[AMQ and MQTT]
+* https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-a-mq-image/[Realtime Decision Server]
+* https://zeppelin.apache.org/[Apache Zeppelin]
+* Simulated Software Sensor - Need a description
```
Comment (sabre1041) [inline README.adoc]: Idea for a description: Spring boot based software sensor for simulating metrics

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+== Prerequisites
+
+* Git
+* Access to an OpenShift environment
+
+_Note: The OpenShift environment can be enabled by CDK platform. To learn how to install CDK, check out https://github.com/redhatdemocentral/cdk-install-demo[CDK-Install Demo]_
+
+* OpenShift Command Line Interface (CLI) - Do we still need this?
```
Comment (sabre1041) [inline README.adoc]: The CLI is needs to be installed wherever the `init.sh` script is launched from

File: README.adoc [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+The https://github.com/sabre1041/iot-ose/blob/master/init.sh[init.sh] script, provided with the project, automates the provisioning process for the demo.
+
+*Step 3:* Execute the init script
+
+ [user@localhost ~]$ ./init.sh
+ 
+The init script triggers asynchronous builds and deployments of A-MQ, Decision Server, Fuse Integration Service, PostgreSQL, Zeppelin and Software Sensor components. OpenShift web console provides a graphical UI into the setup.
+
+*Step 4:* Login to the OpenShift web console using a web browser at 10.1.2.2:5443 (login/password - admin/admin) and select the **iot-ose** project
```
Comment (sabre1041) [inline README.adoc]: This assumes that you are using the CDK. I would genericize to be "Launch the OpenShift web console and login..."

File: README.adoc [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+image:/images/OSE-projects.png[width="500", height="400", align="center"]
```
Comment (sabre1041) [inline README.adoc]: Images are not appearing. Overview diagram is though

File: README.adoc [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+ [user@localhost ~]$ git clone https://github.com/sabre1041/iot-ose.git
+
+
+The https://github.com/sabre1041/iot-ose/blob/master/init.sh[init.sh] script, provided with the project, automates the provisioning process for the demo.
+
+*Step 3:* Execute the init script
+
+ [user@localhost ~]$ ./init.sh
+ 
+The init script triggers asynchronous builds and deployments of A-MQ, Decision Server, Fuse Integration Service, PostgreSQL, Zeppelin and Software Sensor components. OpenShift web console provides a graphical UI into the setup.
```
Comment (sabre1041) [inline README.adoc]: init script now tracks the builds synchronously. Can you update?

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Really good writeup! A couple small changes and it should be good to go

File: README.adoc (hunk 1 above)
Comment (ishuverma) [reply README.adoc]: done

File: README.adoc (hunk 2 above)
Comment (ishuverma) [reply README.adoc]: I was thinking of description of what the sensor does: generates temp, gps and vibration data to simulate the movement of a package on a truck. 
I can write this up and you can review to see if it makes sense

File: README.adoc (hunk 3 above)
Comment (ishuverma) [reply README.adoc]: ok

File: README.adoc (hunk 6 above)
Comment (ishuverma) [reply README.adoc]: done

File: README.adoc (hunk 4 above)
Comment (ishuverma) [reply README.adoc]: to clarify, should I delete: 
"using a web browser" 
or
"using web browser at using a web browser10.1.2.2:5443"

File: README.adoc (hunk 5 above)
Comment (ishuverma) [reply README.adoc]: none of the images are appearing? 

I see them on my branch though? I wonder if they made it through the PR

File: README.adoc (hunk 4 above)
Comment (sabre1041) [reply README.adoc]: I would remove the reference to 10.1.2.2:8443

File: README.adoc (hunk 5 above)
Comment (sabre1041) [reply README.adoc]: The images are part of the PR, but they are not showing up on the README.adoc. The image that is appearing uses the syntax starting with `image::images` denoting a relative path vs  `image:/images` which is not working

Comment (ishuverma) [reply README.adoc]: updated the path for rest of the images. Let me know if it worked

File: README.adoc [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+image::images/OSE-projects.png[width="500", height="400", align="center"]
+
+== Configure the Zeppelin visualization tool
+
+The data visualization is stored in Notebooks. The interpreters allows Zeppelin to connect with Postgresql-backend.
+
+*Step 5:* Locate the *Zeppelin* service on the overview page and click on its url to launch the Zeppelin web console
+
+image:/images/Zeppelin-service.png[width="500", height="400", align="center"]
```
Comment (sabre1041) [inline README.adoc]: This image is not correctly linked

File: README.adoc [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+The https://github.com/sabre1041/iot-ose/blob/master/init.sh[init.sh] script, provided with the project, automates the provisioning process for the demo.
+
+*Step 3:* Execute the init script
+
+ [user@localhost ~]$ ./init.sh
+ 
+The init script triggers synchronous builds and deployments of A-MQ, Decision Server, Fuse Integration Service, PostgreSQL, Zeppelin and Software Sensor components. OpenShift web console provides a graphical UI into the setup.
+
+*Step 4:* Login to the OpenShift web console using a web browser at 10.1.2.2:5443 (login/password - admin/admin) and select the **iot-ose** project
```
Comment (sabre1041) [inline README.adoc]: Still need to remove `10.1.2.2` addresses to genercize

**Review by sabre1041 — APPROVED** [review]

**Pull request merged**